app/models/host_aggregate.rb

```diff
@@ -0,0 +1,43 @@
+class HostAggregate < ApplicationRecord
+  include NewWithTypeStiMixin
+  include Metric::CiMixin
+  include EventMixin
+  include ProviderObjectMixin
+  include VirtualTotalMixin
+
+  acts_as_miq_taggable
+
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+
+  # FIXME: I don't think this is needed here
```

Comment:
@tzumainn @Loicavenel Availability Zones are an obvious one to start with if we're mainly concerned with VMs. In fact, since Host Aggregates can be associated 1-to-1 with availability zones I imagine you may want the same rollups. I just don't know if we need to maintain an explicit VMs association for this or if we can just pull VMs from the VMs on the hosts. If we don't have to have hardware-level rollups I can just follow the Availability Zones model.